src/components/Routes/Routes.jsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
 
-import { HomePage, ProductListingPage, WishlistPage, CartPage, LogInPage, LogOutPage, SignUpPage } from "../../pages/index";
+import { HomePage, ProductListingPage, ProductDetailPage, WishlistPage, CartPage, LogInPage, LogOutPage, SignUpPage } from "../../pages/index";
```

Comment:
Nitpick: While importing from the `index` file of a directory you need not mention `/index` in the path.

Reply:
okk will remember it!!